health-sdk/example-app/src/main/java/net/gini/android/health/sdk/exampleapp/review/ReviewActivity.kt

```diff
@@ -114,14 +113,14 @@ class ReviewActivity : AppCompatActivity() {
                 }
             }
 
-            override fun onPayInvoiceClicked(paymentComponentViewIdentifier: String) {
+            override fun onPayInvoiceClicked(documentId: String) {
                 // Get and show the payment ReviewFragment for the document id in paymentComponentViewIdentifier
```

Comment:
[Request] please take a look at this comment. I don't know if keeping the `paymentComponentViewIdentifier` in the comment is intentional or not. Thanks :-)

Reply:
Thank you for catching that! I will update the comment.